src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -114,6 +125,10 @@ public SendLetterController(
 
     @PutMapping(path = "/{id}/is-failed")
     @ApiOperation(value = "Update failed status when letter was sent to dead letter queue")
+    @ApiResponses({
+        @ApiResponse(code = 401, message = "Invalid service authorisation")
```

Comment:
SonarQube may complain about repeated string

Reply:
Good point.It will be good if we could run our PR against local SONAR instance so issues can be identified earlier in the CD process.

Reply:
Didn't I see Sonarqube check somewhere in PRs?

Reply:
I meant running `./gradlew sonar` locally :) if that makes sense.Of course you need a local instance of SONAR running(which is just docker container)